src/metric_config.cpp

```diff
@@ -18,11 +18,13 @@
 #include <regex>
 #include <string>
 
+#include <boost/algorithm/string/trim.hpp>
```

Comment:
Do we need boost? We have our own string_utils.hpp for trimming

Reply:
Removed.